Wikipedia/Code/NSDictionary+WMFExtensions.m

```diff
@@ -0,0 +1,13 @@
+
+#import "NSDictionary+WMFExtensions.h"
+
+@implementation NSDictionary (WMFExtensions)
+
+- (BOOL)containsNullObjects {
```

Comment:
should this be recursive? so if a value in a dictionary is also a dictionary, it should return TRUE if that contains NSNull